src/main/java/org/spongepowered/api/entity/ai/task/AITask.java

```diff
@@ -95,4 +95,16 @@
      * @return True if can be interrupted, false if not
      */
     boolean canBeInterrupted();
+
+    /**
+     * Resets this task.
+     */
+    void reset();
+
+    /**
+     * Gets the mutex bits. When the | operation result of two mutex bits is 0, they can run
+     * concurrently.
+     * @return The mutex bits which categorizes this task
+     */
+    int getMutexBits();
```

Comment:
So are we going to replace mutex bits with [this catalog type](https://github.com/SpongePowered/SpongeAPI/pull/1012/files#diff-c108db2d21314ed0b6196b9614fd0dc6R31)?